### PR TITLE
Migrate solution to .net core 3.0 for LINQPad 6

### PR DESCRIPTION
**Pull request by lvond**

### CODE REVIEW

File: Src/CsvLINQPadDriver/ConnectionDialog-NoWPF.cs [hunk 1]
```diff
@@ -1,21 +1,16 @@
-﻿using System;
```
Comment (lvond) [inline Src/CsvLINQPadDriver/ConnectionDialog-NoWPF.cs:1]: I wasn't really sure about changes in this file. I just made unnecessary fixes to fix the compilation.

Comment (lvond) [reply Src/CsvLINQPadDriver/ConnectionDialog-NoWPF.cs:1]: Just realized this file wasn't compiled before at all, so the changes are wrong I should rather exclude this file from the library compilation, right?

File: Src/CsvLINQPadDriver/CsvLINQPadDriver.csproj [hunk 2]
```diff
@@ -1,118 +1,56 @@
...
-    <Compile Include="SchemaBuilder.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
+    </Content>
+    <Content Include="FailedConnection.png">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
   </ItemGroup>
+
   <ItemGroup>
     <None Include="packages.config" />
```
Comment (lvond) [inline Src/CsvLINQPadDriver/CsvLINQPadDriver.csproj]: packages.config should go away

File: Src/CsvLINQPadDriver/SchemaBuilder.cs [hunk 3]
```diff
@@ -75,28 +71,21 @@ internal static List<ExplorerItem> GetSchemaAndBuildAssembly(ICsvDataContextDriv
...
-            if (!results.Errors.HasErrors)
-                return null;
-
-            return results.Errors.OfType<CompilerError>().Where(e => !e.IsWarning).Select(e => string.Format("{0},{1}: {2}", e.Line, e.Column, e.ErrorText)).ToArray();
+            var referencedAssemblies = DataContextDriver.GetCoreFxReferenceAssemblies().Concat(new []
+            {
+                typeof(SchemaBuilder).Assembly.Location,
+                typeof(CsvHelper.CsvReader).Assembly.Location
+            }).ToArray();
+            var result = DataContextDriver.CompileSource(new CompilationInput()
```
Comment (lvond) [inline Src/CsvLINQPadDriver/SchemaBuilder.cs:81]: We could probably use Roslyn explicitly, but linqpad DataContextDriver helper provides this functionality as well

File: Tools/CsvLINQPadFileOpen/CsvLINQPadFileOpen.csproj [hunk 4]
```diff
@@ -44,32 +14,23 @@
...
   </ItemGroup>
   <ItemGroup>
     <EmbeddedResource Include="..\..\README.md">
       <Link>DriverBinaries\README.md</Link>
     </EmbeddedResource>
-    <None Include="App.config" />
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Include="..\..\Deploy\header.xml">
-      <Link>DriverBinaries\header.xml</Link>
```
Comment (lvond) [inline Tools/CsvLINQPadFileOpen/CsvLINQPadFileOpen.csproj:71]: header file is no longer required

**Comment by sebastien-f** [comment]
@lvond are you sure it's working ? I just tried your version, it compiles fine. I then zipped and renammed the zip extension to lpx6 to install it. The driver seems to load just fine, it finds the old connections back, but it's impossible to add a new one (can't specify a path, it defaults to c:\*.csv).

**Comment by dobrou** [comment]
@lvond thanky you very much for help with LINQPad6 migration.
I'll check this PR, merge it and add official nuget for installation soon.

**Comment by lvond** [comment]
> @lvond are you sure it's working ? I just tried your version, it compiles fine. I then zipped and renammed the zip extension to lpx6 to install it. The driver seems to load just fine, it finds the old connections back, but it's impossible to add a new one (can't specify a path, it defaults to c:*.csv).

By impossible do you mean the textbox is not editable so you cannot edit the default path? Is at least Drag&Drop working for you? I had no problem with both of ways.

**Comment by sebastien-f** [comment]
@lvond I don't even get the connexion properties window. Here is what I do and what I get : 

1.  Start LinqPad 6 (
2. Click on "Add connection"
3. "Choose Data Context" window opens, select "CSV Context Driver" v1.6.7443.20506 (its location being LocalDataFolder)
4. Click "Next >" at the bottom right of "Choose Data Context" window 
5. Small "Loading..." popup appears, then disappears
6. New connexion with what I guess are default parameters appears in the connexion list with the CSV icon and "c:\*.csv" as connexion name. When expanded, it shows "No files found." in purple. 

If I try to edit that connexion, or any prior connexion no connexion property window appears at all either. Only way I found to add or edit connexions is to launch LP 5, do the job there and then restart LP6 where it's automaticaly imported.

Note that I can query any connexion imported from LP5 with LP6. I just don't get the Connexion Property window.

**Comment by dobrou** [comment]
Thank you again @lvond !
I'm going to merge the PR, solve remaining issues and publish official nuget for the plugin.

**Pull request merged**